cypress/integration/example-jsonschema-form_spec.js

```diff
@@ -0,0 +1,9 @@
+describe('src/js/example-jsonschema-form.html', () => {
+  it('should render -1.00', () => {
+    cy.visit('http://localhost:8000/src/js/example-jsonschema-form.html');
+    cy.get('input[id=root_epsilon]').type('{selectall}0.1');
```

Comment:
I cannot see this input field defined anywhere else. I will need some explanation here.

Reply:
Comment added